deploy/02_sources.ts

```diff
@@ -530,14 +530,8 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
         [
           futureOperatorAddress,
           generateInitCode(
-            ['address', 'address', 'address', 'address', 'address'],
-            [
-              futureBridgeProxyAddress,
-              futureHolographAddress,
-              futureHolographInterfacesAddress,
-              futureRegistryProxyAddress,
-              futureHlgAddress,
-            ]
+            ['address', 'address', 'address', 'address'],
```

Comment:
@alexanderattar I think there is older deploy scripts code in your commits. These changes would remove additional data for contracts in deployments and break everything. You might need to check the `experimental` branch and copy from it the contents of the `deploy/02_sources.ts` file.